ccf_transaction_processor/scripts/register_enclave_attestation_verification_policy.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import http
+import os
+import sys
+import toml
+from urllib.parse import urlparse
+
+from ccf.clients import Identity
+from ccf.clients import CCFClient
+
+from utils import parse_ledger_url
+# pick up the logger used by the rest of CCF
+from loguru import logger as LOG
```

Comment:
We usually import the `logging` package. It would be good to maintain consistency.

Reply:
thank you, As noted above, CCF installation currently does not assume PDO.python installation.